### ISSUE TITLE
Error with compute_bn_statistics.py

### ISSUE THREAD

**Issue by Almo1982** [alexgkendall/caffe-segnet#62]
Hi all,

i trained a couple of networks with segnet so far and everything worked well. Now i have a dataset that i resized to 512x256 pixels. The ground truth are labeled with 19 clases (pixel value 0-18) and pixel value 255 is set as ignore label!
Training works fine, but now when i run the compute_bn_statistics.py script i get the error "Segmentation fault (core dumped)"...i dont understand, it worked for me all the time before. is i take some older caffemodels and training nets the script runs fine?

What could be the problem in this case? i already run this script with training data of 512x256, it can not be the different resolution?! There is only the number of classes i changed and set the ignore label, but what has this to do with these parameters? I already had to change the num_output of the last convolutional layers to another value than the segnet tutorial before and everything worked with the  compute_bn_statistics.py script.

What am i doing wrong? I dont do anything different than all the times before...please help!

Thank you in advance.

Regards
Stefan

**Comment by Almo1982** [comment]
If anybody else has this Problem: fixed it by removing the two last layers (softmax and accuracy). These seem not to be needed when calculating bn_statistics.

Works for me!

**Comment by Jinming-Su** [comment]
Thanks @Almo1982
